web/src/components/Navigation.vue

```diff
@@ -9,7 +9,14 @@
         <v-list-item-title>Help</v-list-item-title>
       </v-list-item-content>
     </v-list-item>
-    <v-list-item link dense exact :to="{ name: 'search', query: { q: '' } }">
+    <v-list-item
+      link
+      dense
+      exact
+      :style="onShiftPressed"
```

Comment:
why do we have changed handling for this item? it seems useless to add the "all" filter

Reply:
yeah true, i'll delete that